modules/library/jdbc/src/main/java/org/geotools/jdbc/JDBCDataStore.java

```diff
@@ -3285,6 +3290,9 @@ private Integer findVarcharColumnLength(AttributeDescriptor att) {
 
                     if (sqlType == sqlTypes[i]) {
                         sqlTypeNames[i] = sqlTypeName;
+                        // Geos-7533
```

Comment:
Please refer to Geotools tickets, not geoserver ones

Reply:
fixed